### PR TITLE
some fix on default configuration

### PR DESCRIPTION
**Pull request by jackzipu**
Fixed the following parts:

1. Compile with SDK 2.2.1, the custom op erf and rsqrt have problem.
    - Comment erf.cc out in CMakeList, as we are using the erf in poplib now
    - Fixed the problem of rsqrt.cc
2. Fixed the problem of default setting of ipu_num, batch_per_step etc set by popart_config.cc were used, which caused the parameters set did not take effect.
    - Make the popart_config.cc set only take effect for pipeline to check whether the configuration consistent or not.
3. Fixed the problem of multiple device detach called, or device detach called during the pipeline is executing, which will cause the poplar exception or even exit.
    - Add status checking for the computation destroy call, make the multiple call in sequence. And make it wait for the pipeline stage terminate(usually it takes less than 1s, it depends on the configuration of batches per step)

### CODE REVIEW

File: ODLA/platforms/odla_popart/odla_compute.cc [hunk 1]
```diff
@@ -112,11 +112,6 @@ odla_status odla_DestroyContext(odla_context ctx) {
 
 odla_status odla_DestroyComputation(odla_computation comp) {
   comp->mark_done();
-  if (comp->session != nullptr) {
```
Comment (weimingzha0) [inline ODLA/platforms/odla_popart/odla_compute.cc:115]: no clean up?
so when do we free up device?

**Comment by jackzipu** [comment]
Hi @weimingzha0 , the PR passed the test, please help to merge it to master. Thanks.

**Comment by weimingzha0** [comment]
> Hi @weimingzha0 , the PR passed the test, please help to merge it to master. Thanks.

Hi @jackzipu Thanks. I have a question (see comments above) regarding device free.

**Review by jackzipu — COMMENTED** [review]
> > Hi @weimingzha0 , the PR passed the test, please help to merge it to master. Thanks.
> 
> Hi @jackzipu Thanks. I have a question (see comments above) regarding device free.

Any new comments?

**Comment by jackzipu** [comment]
Close this PR as it's merged with #543

**Pull request closed without merging**